aries-site/src/examples/templates/wizard/WizardValidationExample.js

```diff
@@ -0,0 +1,346 @@
+  return (
+    <>
+      <Box gap="small" margin={{ bottom: 'large' }}>
+        <List data={data} pad={{ horizontal: 'none', vertical: 'small' }}>
+          {(datum, index) => (
+            <Box key={index} direction="row" gap="medium" align="center">
+              <Checkmark color="text-strong" size="small" />
+              <Text color="text-strong">{datum}</Text>
+            </Box>
+          )}
+        </List>
+        <Text color="text-strong">
+          Include guidance to what will occur when “Finish Setup” is clicked.
+        </Text>
+      </Box>
+      <Button fill="horizontal" label="Finish Setup" primary type="submit" />
+    </>
+  );
+};
+
+const steps = [
+  {
+    description: `Step one description. Keep each step simple and in chunks 
+    easy enough to fit on a single page.`,
+    inputs: (active, setActive, formValues) => (
+      <StepOne active={active} setActive={setActive} formValues={formValues} />
+    ),
+  },
+  {
+    description: 'Step 2 description.',
+    inputs: (active, setActive) => (
+      <StepTwo active={active} setActive={setActive} />
+    ),
+  },
+  {
+    description: 'Provide a summary of what was accomplished or configured. ',
+    inputs: (active, setActive) => (
+      <StepThree active={active} setActive={setActive} />
+    ),
+    title: 'Finish title',
+  },
+];
+
+export const WizardValidationExample = () => {
+  const [active, setActive] = useState(1);
+
+  // store form values in state so they persist
+  // when user goes back a step
+  const [formValues, setFormValues] = useState(defaultFormValues);
+
+  // controls state of cancel layer
+  const [open, setOpen] = useState(false);
+  const size = useContext(ResponsiveContext);
+
+  return (
+    <>
+      <Box width={{ max: 'xxlarge' }} margin="auto" fill>
+        <WizardHeader active={active} setActive={setActive} setOpen={setOpen} />
+        <Box
+          align="center"
+          pad={size !== 'small' ? 'large' : 'medium'}
+          flex={false}
+        >
+          <Box width="medium" gap="medium">
+            <Box gap="medium" flex={false}>
+              <Box>
+                <Heading color="text-strong" margin="none">
+                  {steps[active - 1].title || `Step ${active} Title`}
+                </Heading>
+                <Text
+                  color="text-strong"
+                  size="small"
+                >{`Step ${active} of ${steps.length}`}</Text>
+              </Box>
+              <Text color="text-strong" size="large">
+                {/* Index an array starts at 0 */}
+                {steps[active - 1].description}
+              </Text>
+            </Box>
+            <Form
+              value={formValues}
+              onChange={nextValue => setFormValues(nextValue)}
+              onSubmit={({ value }) => console.log(value)}
+            >
+              {/* Index an array starts at 0 */}
+              {steps[active - 1].inputs(active, setActive, formValues)}
```

Comment:
If we used the term `activeIndex` we could initialize state at 0 (as opposed to 1). Then we wouldn't need to worry about all of the `active - 1`. Could eliminate the need for code comments, unnecessary operations, and probably less prone to bugs.

Reply:
Would require changes throughout each reference to active, but it feels cleaner.

Reply:
There would still need to me comments and calculations in the areas where we are using the active state to display the step number. in these places, we would do `active + 1` instead.

Reply:
I am open to either direction, but feel like some calculation/comments will be necessary in either direction.